CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
+
```

Comment:
Please revert all whitespace changes in this file, especially the trailing whitespace in link section at the bottom